src/System.Net.Http/tests/FunctionalTests/ResponseStreamTest.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Net.Tests;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class ResponseStreamTest
+    {
+        private readonly ITestOutputHelper _output;
+        
+        public ResponseStreamTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public async Task GetStreamAsync_ReadToEnd_Success()
+        {
+            var client = new HttpClient();
+            
+            var stream = await client.GetStreamAsync(HttpTestServers.RemoteGetServer);
+            using (var reader = new StreamReader(stream))
+            {
+                var responseBody = reader.ReadToEnd();
+                _output.WriteLine(responseBody);
```

Comment:
I'll add some simple validations for this.  

But my plan is to do it better in future PR's with a SHA1 hash of the expected response body content and include that hash as a special HTTP response header.  That is how it is usually done in the real world to confirm getting an unaltered payload back.